test/export-api-def.test.js

```diff
@@ -55,83 +55,87 @@ describe('loopback:export-api-def generator', function() {
+          'application/xml', 'text/xml',
+        ]);
+        expect(content.produces).to.have.members([
+          'application/json',
+          'application/xml', 'text/xml',
+          'application/javascript', 'text/javascript',
+        ]);
+      });
   });
 
-  it('produces YAML format', function(done) {
-    var gen = givenGenerator();
-    gen.options.output = 'swagger.yaml';
-
-    gen.run(function() {
-      var content = readYamlFile();
-      expect(content).to.have.property('swagger', '2.0');
-      expect(content).to.not.have.property('host');
-      expect(content).to.not.have.property('schemes');
-      expect(content).to.not.have.property('public');
-      expect(content).to.have.property('basePath');
-      expect(content).to.have.property('info');
-      expect(content.info).to.have.property(
-        'title', 'test-app'
-      );
-      expect(content).to.have.property('tags');
-      expect(content.consumes).to.have.members([
-        'application/json',
-        'application/x-www-form-urlencoded',
-        'application/xml', 'text/xml',
-      ]);
-      expect(content.produces).to.have.members([
-        'application/json',
-        'application/xml', 'text/xml',
-        'application/javascript', 'text/javascript',
-      ]);
-      done();
-    });
+  it('produces YAML format', function() {
+    return helpers.run(path.join(__dirname, '../export-api-def'))
+      .cd(SANDBOX)
+      .withOptions({
+        output: 'swagger.yaml',
+      }).then(function() {
+        var content = readYamlFile();
+        expect(content).to.have.property('swagger', '2.0');
+        expect(content).to.not.have.property('host');
+        expect(content).to.not.have.property('schemes');
+        expect(content).to.not.have.property('public');
+        expect(content).to.have.property('basePath');
+        expect(content).to.have.property('info');
+        expect(content.info).to.have.property(
+          'title', 'test-app'
+        );
+        expect(content).to.have.property('tags');
+        expect(content.consumes).to.have.members([
+          'application/json',
+          'application/x-www-form-urlencoded',
+          'application/xml', 'text/xml',
+        ]);
+        expect(content.produces).to.have.members([
+          'application/json',
+          'application/xml', 'text/xml',
+          'application/javascript', 'text/javascript',
+        ]);
+      });
   });
 
   describe('running on project with onging async operations', function() {
-    it('immediately does not exit the process', function(done) {
-      var gen = givenGenerator();
+    it('immediately does not exit the process', function() {
       expect(PROCESS_EXIT_STUB.called).to.equal(false);
-      gen.run(function() {
-        expect(PROCESS_EXIT_STUB.called).to.equal(false);
-        done();
-      });
+      return helpers.run(path.join(__dirname, '../export-api-def'))
+        .cd(SANDBOX)
+        .withOptions({
+          output: 'swagger.yaml',
```

Comment:
I don't see this option in the previous code. Any reason we're introducing it here?